tests/components/synology_dsm/conftest.py

```diff
@@ -4,10 +4,20 @@
 from tests.async_mock import patch
 
 
+def pytest_configure(config):
+    """Register custom marker for tests."""
+    config.addinivalue_line(
+        "markers", "noautofixt: mark test to disable bypass_setup_fixture"
```

Comment:
Maybe rename the marker to `no_bypass_setup`.

Reply:
ups ... I was so happy that it works now, that I have forgotten to think about the name of the marker 🤣